components/brave_rewards/resources/rewards_panel/components/panel.tsx

```diff
@@ -84,11 +133,15 @@ export function Panel () {
   }
 
   return (
-    <div>
-      <div className='rewards-panel' data-test-id='rewards-panel'>
-        {userType !== 'unconnected' ? renderFull() : <LimitedView />}
-      </div>
-      <PanelOverlays />
-    </div>
+    onboardingResult || !rewardsEnabled || needsCountry
```

Comment:
Nit: consider whether an early return might be more readable here, e.g.

```tsx
if (onboardingResult || !rewardsEnabled || needsCountry) {
  return renderOnboarding()
}
```

Reply:
Updated in https://github.com/brave/brave-core/commit/1bb3417cf7a2cf42118a77eddba9630547cb4692